www/~/%username/subscriptions.json.spt

```diff
@@ -0,0 +1,100 @@
+""" Get or change authenticated user's subscriptions.
+"""
+from aspen import Response
+from gratipay.models.team import Team
+from gratipay.utils import get_participant
+
+[-----------------------------------------------------------------------------]
+
+def format_subscription(s):
+    return {
+        "team_name": s.team_name,
+        "team_slug": s.team_slug,
+           "amount": str(s.amount),
+              "due": str(s.due),
+            "ctime": s.ctime,
+            "mtime": s.mtime
+    }
+
+def format_error(team_slug, error):
+    return {
+        "team_slug": team_slug,
+            "error": error
+    }
+
+participant = get_participant(state, restrict=True)
+
+if request.method == 'GET':
+
+    # Fetch all subscriptions for this user.
+    subscriptions, totals = participant.get_giving_for_profile()
+
+    slug = request.qs.get("team_slug", "")
+
+    if slug: # Filter subscription info only for this team from the list.
```

Comment:
How is this intended to related to the existing [`%team/payment-instruction.json.spt`](https://github.com/gratipay/gratipay.com/blob/1939/www/%25team/payment-instruction.json.spt) API? Looks like it at least partially duplicates the functionality over there. Why?